test/core/end2end/end2end_test_suites.cc

```diff
@@ -550,7 +550,29 @@ class InsecureFixtureWithPipeForWakeupFd : public InsecureFixture {
 };
 #endif
 
+std::string GetTempDir(){
+#ifdef WIN32
+  // windows temp dir usually exceeds uds length.
+  // std::array<char, MAX_PATH + 1> buff = {};
+  // DWORD len = GetTempPathA(buff.size(), buff.data());
+  // if (len == 0) {
+  //   GPR_ASSERT(false);
+  // }
+  // return std::string(buff.data(), len);
+  std::string temp_dir = "C:/tmp/";
```

Comment:
May need to find a better way to use the temp dir on windows.
Currently the test makes a dir from the root of the c drive.

The temp dir should be "C:\Users\user1\AppData\Local\Temp\grpc_fullstack_test.xxx.xxx" which exceeds uds name length.

Reply:
This may work for our test suite. If so, it may be sufficient to add a TODO to improve later if we need to. I'll kick off the tests shortly.